sdk/pinocchio/src/sysvars/tests/slot_hashes_tests.rs

```diff
@@ -0,0 +1,855 @@
+#[cfg(test)]
```

Comment:
Should this be

```suggestion
#![cfg(test)]
```

This annotation currently applies only to the `use super::*` import, not to the `tests` submodule.

Reply:
As it was, the import of mod tests in the main slot_hashes.rs file:

```
#[cfg(test)]
mod slot_hashes_tests;
```

gated the whole module.

But reorganizing the tests in my next push will change this.